skopt/optimizer/optimizer.py

```diff
@@ -153,9 +156,13 @@ def __init__(self, dimensions, base_estimator="gp",
         else:
             self.cand_acq_funcs_ = [self.acq_func]
 
+        if self.acq_func == "noisyEI":
+            self.noisyEI_N_variants = noisyEI_N_variants
```

Comment:
Sooner or later I will learn to use git propery......
Those 2 lines can be removed, I'm setting it on line 165